openquake/hazardlib/gsim/projects/acme_2019.py

```diff
@@ -330,35 +385,91 @@ def get_acc_from_disp(self, disp, imt):
         acc = np.log(disp * (2 * np.pi / imt)**2)
         return acc
 
-    def get_mean_and_stddevs(self, sites, rup, dists, imt, stds_types):
+    def extrapolate_in_PSA(self, sites, rup, dists, imt_high, 
+                           set_imt, stds_types, imt):
+
+        extrap_mean = []
+        t_log10 = np.log10([im for im in set_imt])
+        from openquake.hazardlib.contexts import DistancesContext
+        for d in np.arange(0,len(dists.rjb)):
+            dist = DistancesContext()
+            if hasattr(dists, 'rjb'):
+                dist.rjb = np.array([dists.rjb[d]])
+            if hasattr(dists, 'rrup'):
+                dist.rrup= np.array([dists.rrup[d]])
+            means_log10 = []
+            for im in set_imt:
+                mean_ln, _ = self.gmpe.get_mean_and_stddevs(
+                                   sites, rup, dist, SA(im), stds_types)
+                mean = np.exp(mean_ln[0])
+                means_log10.append(np.log10(mean))
+                
+            mb = np.polyfit(t_log10, means_log10, 1)
+            mean_imt_log10 = mb[0] * np.log10(imt) + mb[1]
+            extrap_mean.append(np.log(10**mean_imt_log10))
+            _ = np.zeros_like(extrap_mean)
```

Comment:
Bad name, "_" means that the variable is not used, but then you return it in the line below, so please give it a meaningful name. On the other hand, if it is not needed, completely remove it and do not return it.

Reply:
Actually this should be deleted (I've done so now)